packages/nextjs/pages/api/etl/index.ts

```diff
@@ -4,55 +4,61 @@ import type { NextApiRequest, NextApiResponse } from "next";
 import { OnchainMetricsByProject } from "~~/app/types/OSO";
 import dbConnect from "~~/services/mongodb/dbConnect";
 import ETLLog from "~~/services/mongodb/models/etlLog";
-import GlobalScore from "~~/services/mongodb/models/globalScore";
-import Metric, { MetricNames, Metrics } from "~~/services/mongodb/models/metric";
+import GlobalScore, { TempGlobalScore } from "~~/services/mongodb/models/globalScore";
+import Metric, { Metrics } from "~~/services/mongodb/models/metric";
 import Project from "~~/services/mongodb/models/project";
-import ProjectMetricSummary from "~~/services/mongodb/models/projectMetricSummary";
-import ProjectScore, { IProjectScore } from "~~/services/mongodb/models/projectScore";
+import ProjectMetricSummary, { TempProjectMetricSummary } from "~~/services/mongodb/models/projectMetricSummary";
+import ProjectScore, { IProjectScore, TempProjectScore } from "~~/services/mongodb/models/projectScore";
+
+export const maxDuration = 300;
+export const runtime = "nodejs";
```

Comment:
Also I don't think this gets used anywhere

Reply:
These are used by Vercel. By defining them here it knows that this function uses nodejs runtime and is allowed to run the maximum 300 seconds. I should add a comment though.